finder-app/Makefile

```diff
@@ -0,0 +1,11 @@
+# Makefile for cross compilation and native compilation 
+# Author : Sachin Mathad
+
+cc=gcc
```

Comment:
@sachininja convention is to use capital `CC` for this - see https://www.gnu.org/software/make/manual/html_node/Implicit-Variables.html